### ISSUE TITLE
Accessibility statement including a named contact

### ISSUE THREAD

**Issue by ailsaleen** [WeAreSnook/human-services#74]
-

**Comment by UsmanAfzal** [comment]
@ailsaleen I'm producing a draft for this to include mandatory content but will be complete once the full audit is done on the test environment.

@anandamaryon I haven't identified this in the current wireframes but to confirm that there is no intention for the following:

- Do we expect to have any documents such as PDF?
- Any tabular data?

**Comment by ailsaleen** [comment]
Sounds good for the first point, thanks Usman!

To your second points:

- No current plans for a PDF but one user did suggest a factsheet that people could circulate with their org, so it could go on the 'backlog' that we hand off to the client. 
- FYI we are also considering working with the creator of the video from the US version (openreferral.org) to create a UK video - so there could be some accessibility implications there.
- For tabular data, @chiayingt is working on an adoption dashboard here: https://www.figma.com/proto/1qiA6bdGXhAoy2Ex5GUUsE/ORUK-wireframes-0.2?node-id=789%3A3119&viewport=396%2C83%2C0.059872932732105255&scaling=min-zoom&page-id=680%3A0 

I'm wondering if we need some separate accessibility document for the client for changes they may do in future without us (this might be separate to this ticket) - best practices? Or to include accessibility implications in each of the 'suggested features' backlog that we'll hand off.
 
And we might want to weave some accessibility best practices into the Comms & Engagement pack that Robyn is working on here (Shall I add this as a separate ticket?): https://docs.google.com/document/d/19dxhzUgVrRzIz7qt4RiPatb316vKIcRLujiWgmAYbq8/edit?disco=AAAAH9kBMVQ&ts=603f9dc5&usp_dm=false

**Comment by UsmanAfzal** [comment]
- Once the ticket is created for the video, i'll add in any requirements for that.
- I've added details regarding the dashboard design
- If we have a collated list of backlog items I can include a list of things that should be considered.
- Regarding comms and engagement it would be easier to include requirements on separate tickets rather than a single place as it is dependent on the given medium.

**Comment by UsmanAfzal** [comment]
Initial draft based on current understandings and wireframes: https://docs.google.com/document/d/1iTHud59rr1naYmvGnlrcxoubwyjB6bpXpzKJHdwosng/edit?usp=sharing

**Comment by annhelen** [comment]
Hi @ailsaleen (cc @UsmanAfzal)
Have been checking in with Usman about this. I set up a draft accessibility statement last week in the content folder so Usman and I can work together to make sure content, etc, meets WCAG standards and is correctly referenced in the AS.
[Accessibility statement for Open Referral UK_draft 0.1. ](https://docs.google.com/document/d/1XLC5zwxQN2Y2Fu0_QB1oLUo_ITXs2f31aukAp87XVRE/edit#heading=h.bllx43y2mn27)
Could we also set up a ticket for URLs? (Have discussed with Usman and also want to make sure we are compliant around URLs.)
Who should be added to the AS as the person/organisation to contact about accessibility issues? (This is mandatory.)

**Comment by UsmanAfzal** [comment]
@annhelen The version I posted a few days ago has much of the content found in the template removed that will not be necessary for us at this stage. It may be worth starting from there and add on further?
